sdk/attestation/azure-security-attestation/src/samples/com/azure/security/attestation/ReadMeSamples.java

```diff
@@ -0,0 +1,95 @@
+package com.azure.security.attestation;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.test.TestMode;
+import com.azure.security.attestation.models.AttestSgxEnclaveRequest;
+import com.azure.security.attestation.models.AttestationResponse;
+import com.azure.security.attestation.models.AttestationType;
+import com.azure.security.attestation.models.DataType;
+import com.azure.security.attestation.models.JsonWebKeySet;
+import com.azure.security.attestation.models.PolicyResponse;
+import com.azure.security.attestation.models.RuntimeData;
+import com.nimbusds.jose.JOSEObject;
+import com.nimbusds.jwt.JWTClaimsSet;
+import org.junit.jupiter.api.Assertions;
+
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.util.Base64;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ReadMeSamples extends AttestationClientTestBase {
```

Comment:
nit: `ReadMeSamples` -> `ReadmeSamples` and 
       Could add some java doc explaining Readme. 
        An extra line below.